shelf/bulk_update/container.py

```diff
@@ -15,16 +15,16 @@ def __init__(self, config, logger):
         """
         self.config = config
         self.logger = logger
-
         self._runner = None
 
-    @property
-    def runner(self):
+    def create_runner(self, search_update_function):
         """
+            Args:
+                search_update_function(function):   The search update function
```

Comment:
Good call. I was having a hard time coming up with a name... So I just picked that and stuck with it. I'll change it everywhere it is. cff638f